DSharpPlus/Entities/Guild/DiscordGuild.cs

```diff
@@ -249,6 +249,18 @@ public DiscordChannel AfkChannel
         [JsonConverter(typeof(SnowflakeArrayAsDictionaryJsonConverter))]
         internal ConcurrentDictionary<ulong, DiscordRole> _roles = new();
 
+
+        /// <summary>
+        /// Gets a collection of this guild's stickers.
+        /// </summary>
+        [JsonIgnore]
+        public IReadOnlyDictionary<ulong, DiscordMessageSticker> Stickers { get; internal set; }
+
+        [JsonProperty("stickers", NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
This doesn't appear to be documented yet.

Reply:
[I was looking at an open PR](https://github.com/discord/discord-api-docs/pull/3128)

Reply:
Oh alright, I'll wait to merge this when that gets merged then.